CorsixTH/Lua/persistance.lua

```diff
@@ -260,10 +260,13 @@ function SaveGameFile(filename)
   f:close()
 end
 
+--! Loads the game's save into the game
 function LoadGame(data)
   --local status, res = xpcall(function()
   local objtable = MakePermanentObjectsTable(true)
   local state = assert(persist.load(data, objtable))
+  -- Here we want to check the game we're loading is same or older than the current version
+  if not TheApp:checkCompatibility(state.world.savegame_version, state.world.release_version) then return end
```

Comment:
You want to check or are you checking?

If the former, it likely needs a `TODO`, if the latter, the code line seems to quite nicely express what you're aiming for.
(Same or older than current version  is what you actually check, ie it's the implementation of `checkCompatibility`, so that belong more inside the function than above it.)